### ISSUE TITLE
Regression remote importing trivial x/html_escape module starting in 1.36.1

### ISSUE THREAD

**Issue by johnspurlock** [denoland/deno#20211]
To reproduce:
```
$ deno upgrade --version 1.36.1
$ deno eval "import { html } from 'https://deno.land/x/html_escape@v1.1.5/html.ts'"
error: missing field `now` at line 16 column 1
```

Was fine in 1.36.0
```
$ deno upgrade --version 1.36.0
$ deno eval "import { html } from 'https://deno.land/x/html_escape@v1.1.5/html.ts'"
$ <no error>
```

This is a three year old trivial module with no deps that's been working fine for months.  Works fine when copied locally, so not sure exactly what the problem is in 1.36.1.

Thanks!
\- John

**Comment by johnspurlock** [comment]
Also happens with:
```
$ deno eval "import { html } from 'https://cdn.skypack.dev/lit-element@2.5.1'"
```

Maybe something with the html template type?

**Comment by dsherret** [comment]
Fixed by https://github.com/denoland/deno/pull/20192

A release will go out tomorrow.

**Comment by johnspurlock** [comment]
Confirmed fixed in 1.36.2 - thanks!
